nanoFramework.System.Text.Benchmark/StringBenchmark.cs

```diff
@@ -8,116 +8,150 @@
 
 namespace nanoFramework.System.Text.Benchmark
 {
-    [IterationCount(100)]
+
+    // TODO - set iterations back to 100
+
+    [IterationCount(1)]
```

Comment:
I guess you can use a constant for that ;-) so you will have to adjust only once ;-)

Reply:
done